### PR TITLE
Update 2016-12-12-writing-efficient-numerical-code.md

### PR DESCRIPTION
**Pull request by 9il**

### CODE REVIEW

**Comment by 9il** [comment]
ping @ljubobratovicrelja

File: _posts/2016-12-12-writing-efficient-numerical-code.md [hunk 1]
```diff
@@ -341,6 +341,6 @@ growing our young **scientific ecosystem in D**!
 <small><a name="footndslice"></a>[1] [*std.experimental.ndslice*](https://dlang.org/phobos/std_experimental_ndslice.html)
 is the package providing structures and tools for multidimensional data processing. [*mir.ndslice*](https://github.com/libmir/mir#notes)
 is the development version of `std.experimental.ndslice` in [**Mir**](https://github.com/libmir/mir), Generic Numerical Library for Science and Machine Learning. Note that `mir.ndslice` and `std.experimental.ndslice` will
-removed and a new replacement will be provided as solid integrated solution to replace [*ndslice*], [*std.algorithm*], and [*std.range*].
+removed and a new replacement will be provided as solid integrated solution to replace [*ndslice*], [*std.algorithm*], and [*std.range*]. The Future Mir Std library would not need `Yes.vectorized` and `Yes.fastmath` flags.
```
Comment (ljubobratovicrelja) [inline _posts/2016-12-12-writing-efficient-numerical-code.md]: Why wont we need those flags? - should it be mentioned how would we overcome this? I believe the reader would question this.

**Comment by 9il** [comment]
> Why wont we need those flags? - should it be mentioned how would we overcome this? I believe the reader would question this.

I don't think we need to mentioned this. You may if you want.
1. fastmath will good enough to declare only for lambda function
2. vectorization will work out of the box because we will have new kinds of ndslice. The current kind is numpy like tensors: universal, but slow. We will have other kinds too.

**Comment by ljubobratovicrelja** [comment]
> I don't think we need to mentioned this. You may if you want.
> 
> fastmath will good enough to declare only for lambda function
> vectorization will work out of the box because we will have new kinds of ndslice. The current kind is numpy like tensors: universal, but slow. We will have other kinds too.

OK, fair enough. Feel free to merge!

**Comment by 9il** [comment]
> Why wont we need those flags? - should it be mentioned how would we overcome this? I believe the reader would question this.
> I don't think we need to mentioned this. You may if you want.
> 
> fastmath will good enough to declare only for lambda function
> vectorization will work out of the box because we will have new kinds of ndslice. The current kind is numpy like tensors: universal, but slow. We will have other kinds too.

Will you write a forum announce? Also you may want to ask in the forum to improve the article (nitpicks or typos and other) without huge changes. Then you can ask Mike to to share it at reddit (he has big rating) or share it yourself.

**Comment by ljubobratovicrelja** [comment]
> Will you write a forum announce?

Sure.

> Then you can ask Mike to to share it at reddit (he has big rating) or share it yourself.

If its alright, I'd ask Mike for this if he's willing. I'm not that active on reddit.

**Pull request merged**